services/orders/item_add/lambda_function.py

```diff
@@ -8,20 +8,47 @@
 dynamodb = boto3.resource('dynamodb')
 # get the users table
 ORDERS_TABLE = os.environ['ORDERS_TABLE']
-STOCK_TABLE = os.environ['STOCK_TABLE']
+
+# invoke lambda function with given name and payload
+def invoke_lambda(name, payload, invocation_type='RequestResponse'):
+    print(f'invoking lambda function: {name}')
+    client = boto3.client('lambda')
```

Comment:
This client could be shared between requests by initiating it globally, thus saving some overhead.

Reply:
Agree if the implementation is not too complex.

Reply:
As far as I know, the implementation should be a matter of moving the assignment out of the function.

Reply:
done !